app/models/series.rb

```diff
@@ -104,4 +108,23 @@ def schedule_for_episode_number(episode_number)
 
     schedule_index
   end
+
+  def destroy
+    if v4?
+      really_destroy!
```

Comment:
I am still a pretty big fan of paranoid, but I can be convinced.
That seems like a larger debate however, so let's punt on it / log it.
This is consistent with pieces/stories - let's go with consistency for now and readdress closer to a wider release if this is seeming like a pain point still.